src/components/PageSection.tsx

```diff
@@ -4,7 +4,11 @@ type Props = {
   title: string
   description: string
   textPosition: 'left' | 'right'
-  image: string
+  image?: string
+} & typeof defaultProps
+
+const defaultProps = {
```

Comment:
if not using `typeof defaultProps` then can just assign object value to `PageSectionComponent.defaultProps`

Reply:
Fixed in d05127c